robot/src/main/java/us/ilite/robot/modules/Elevator.java

```diff
@@ -0,0 +1,280 @@
+
+package us.ilite.robot.modules;
+
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMax.IdleMode;
+import com.team254.lib.util.Units;
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import com.team254.lib.drivers.talon.TalonSRXFactory;
+
+import us.ilite.common.config.SystemSettings;
+import us.ilite.lib.drivers.SparkMaxFactory;
+
+import edu.wpi.first.wpilibj.Talon;
+import edu.wpi.first.wpilibj.Timer;
+import us.ilite.common.lib.control.PIDController;
+import us.ilite.common.lib.control.PIDGains;
+import com.team254.lib.util.Util;
+import us.ilite.common.Data;
+
+
+public class Elevator extends Module {
+
```

Comment:
This class does not do any logging via Codexes.  Please log this data:
Desired Power
Current Encoder Ticks
Current NEO Encoder Ticks
ALL of the Booleans that determine state (at top, at bottom, etc)
Current
Bus Voltage
The ordinal values of the elevator states (mDesiredPosition, mCurrentPosition, mCurrentState)
(do not worry about logging the time; that is logged automatically)